guac-configuration.md

```diff
@@ -0,0 +1,164 @@
+  - **When to Change**: Change to a secure password for production use.
+
+- **neo4j-addr**: `neo4j://localhost:7687`
+  - **Description**: The address of the Neo4j server.
+  - **When to Change**: Update this if your Neo4j instance is not running locally or is hosted in the cloud.
+
+- **neo4j-realm**: `neo4j`
+  - **Description**: The realm for the Neo4j database, typically used for multi-tenancy.
+  - **When to Change**: Adjust if your setup uses a different realm for authentication.
+
+### Neptune
+- **neptune-user**: `username`
+  - **Description**: The username for connecting to the Neptune database.
+  - **When to Change**: Change this to match your Neptune database credentials.
+
+- **neptune-endpoint**: `localhost`
+  - **Description**: The endpoint for the Neptune database.
+  - **When to Change**: Update this to the actual endpoint if using AWS Neptune.
+
+- **neptune-port**: `8182`
+  - **Description**: The port for the Neptune database.
+  - **When to Change**: Change if your Neptune instance uses a non-default port.
+
+- **neptune-region**: `us-east-1`
+  - **Description**: The AWS region where the Neptune database is hosted.
+  - **When to Change**: Modify to match the region of your AWS Neptune instance.
+
+- **neptune-realm**: `neptune`
+  - **Description**: The realm for the Neptune database.
+  - **When to Change**: Adjust if your setup uses a different realm.
+
+## Pub/Sub Configuration
+
+- **pubsub-addr**: `nats://localhost:4222`
+  - **Description**: The address of the NATS server for pub/sub messaging.
+  - **When to Change**: Update if your NATS server is hosted elsewhere or uses a different port.
+
+- **publish-to-queue**: `true`
+  - **Description**: Whether to publish messages to the queue.
+  - **When to Change**: Set to `false` if you do not want to use queue-based messaging.
+
+## Blob Store Configuration
+
+- **blob-addr**: `file:///tmp/blobstore?no_tmp_dir=true`
+  - **Description**: The address of the blob store.
+  - **When to Change**: Change to use a different storage backend, such as AWS S3 or Google Cloud Storage (examples).
+
+## Certifier Configuration
+
+- **interval**: `20m`
+  - **Description**: The interval at which the certifier runs.
+  - **When to Change**: Adjust based on how frequently you need certification checks.
+
+- **last-scan**: `4`
+  - **Description**: The number of hours since the last scan was run. A value of `0` means the scan will run on all packages/sources.
+  - **When to Change**: Set to `0` for a full scan or adjust based on your scanning frequency needs.
+
+- **certifier-batch-size**: `60000`
+  - **Description**: The batch size for the package pagination query.
+  - **When to Change**: Modify to optimize performance based on your system's capabilities.
+
+- **certifier-latency**: `""`
+  - **Description**: Artificial latency to throttle the certifier.
+  - **When to Change**: Use to introduce delays if needed to manage load.
+
+## Deps.dev Configuration
```

Comment:
Probably better to make this an h3, since the deps.dev certifier is a specific certifier